shared/database/source_control/working_dir.py

```diff
@@ -170,6 +170,7 @@ def list(
         from shared.database.permissions.roles import ValidObjectTypes
 
         # Permissions: get datasets that user can see
+        # TODO: Can we remove this now? 
```

Comment:
Another policy check that I think we can remove, but not sure if depended on elsewhere